spring/spring-boot/pom.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.infinispan</groupId>
+        <artifactId>infinispan-parent</artifactId>
+        <version>12.0.0-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>infinispan-spring-boot-starter-parent</artifactId>
+    <name>Infinispan Spring Boot Starter Parent</name>
+    <packaging>pom</packaging>
+
+    <description>Infinispan Spring Boot Starter parent POM</description>
+    <dependencyManagement>
```

Comment:
`<dependencyManagements>` needs to go away! it is replacing the Infinispan dependency.
actually, it is compiling the starter against Infinispan 10.1.8.Final instead of using 12

Reply:
fixed